rclcpp/include/rclcpp/clock.hpp

```diff
@@ -121,6 +123,16 @@ class Clock
     JumpHandler::post_callback_t post_callback,
     const rcl_jump_threshold_t & threshold);
 
+  /**
+   * Get current time from the time source specified by clock_type as nanoseconds.
+   *
+   * \return RCUTILS_RET_OK if time was successfully retrieved
+   * \throws anything rclcpp::exceptions::throw_from_rcl_error can throw.
+   */
+  RCLCPP_PUBLIC
+  rcutils_ret_t
+  get_now_as_ns(rcutils_time_point_value_t * now);
```

Comment:
@BMarchi nit: why not `get_now_as_timepoint` or even just `timepoint`? The fact that it's just an integer counting nanoseconds is a bit of an implementation detail.

Reply:
Also, should `Clock::now()` use this function instead of duplicating code?

Reply:
A name including `now` and `time_point` would be preferred by me too.